components/invitation_modal/invite_as.tsx

```diff
@@ -103,6 +108,8 @@ export default function InviteAs(props: Props) {
                     onToggle={() => {
                         props.setInviteAs(props.inviteType === InviteType.GUEST ? InviteType.MEMBER : InviteType.GUEST);
                     }}
+                    testId={props.inviteType === InviteType.GUEST ? 'inviteMembersLink' : 'inviteGuestLink'}
+                    id={props.inviteType === InviteType.GUEST ? 'inviteMembersLink' : 'inviteGuestLink'}
```

Comment:
Any opportunities to consolidate the two?

Reply:
yep! This is only used in one other place, so was able to always read from id, and instead pass in a boolean flag if we want to override the patter of the test id being prefixed by `id-`.